src/components/html-injection/ko/widgetEditorView.html

```diff
@@ -0,0 +1,11 @@
+<fieldset class="form flex flex-item flex-item-grow no-overflow">
+    <div class="form-group flex flex-column" style="display: flex; margin-bottom: 0;">
```

Comment:
@azaslonov unfortunately I was unable to find a way how to get rid of this inline style (using only existing classes), do you have any idea or I should just create a new class?

Reply:
Try to take a look at [this editor](https://github.com/paperbits/paperbits-styles/blob/master/src/workshops/styleEditor/styleEditor.html), it has all kinds of cases. If your case still doesn't fit into any of it, then yes, just create a new class.